test/fn.spec.php

```diff
@@ -160,6 +160,14 @@ function _idArgs(...$args) {
             expect(toArray($res))->equal([1,2,3,4]);
         });
     });
+    describe('complement', function() {
+        docFn(complement::class);
+        test('Returns a function that, for the same arguments, returns the same values as its argument function, but negated', function() {
```

Comment:
We need to add a comment describing this is an alias for the curried not behavior so it's clear that these do basically the same things.